python/dask_cudf/dask_cudf/io/parquet.py

```diff
@@ -93,40 +89,35 @@ def _read_paths(
         dataset_kwargs = dataset_kwargs or {}
         dataset_kwargs["partitioning"] = partitioning or "hive"
 
-        # Non-local filesystem handling
-        paths_or_fobs = paths
-        if not _is_local_filesystem(fs):
-            paths_or_fobs = [
-                _fsspec_data_transfer(fpath, fs=fs) for fpath in paths
-            ]
-
         # Use cudf to read in data
         try:
             df = cudf.read_parquet(
-                paths_or_fobs,
+                paths,
```

Comment:
Removing the special-case handling is now possible because read_parquet could always read remotely, but we now have a way of passing the provided `filesystem` object through?

Reply:
Yes, cudf was always (at least for the past two years) able to handle remote file access. However, we didn't have a way to pass through the the remote filesystem object (cudf previously needed to use `storage_options` to re-infer the filesystem itself, and we don't want to track the storage_options in every read task when we are already tracking the filesystem).